index.html

```diff
@@ -2001,9 +2001,29 @@
       <section>
         <h3>Frequency of Claim Issuance</h3>
 
-        <p class="issue">
-The rate at which an issuer issues claims may be a privacy violation.
+        <p>
+As Section <a href="#usage-patterns"></a> details, usage patterns can be
+correlated into certain types of behavior. Part of this correlation is
+mitigated when a <a>holder</a> uses a verifiable credential without the
+knowledge of the <a>issuer</a>. Issuers may defeat this protection by making
+their claims short lived and renewal automatic.
+        </p>
+
+        <p>
+For example, an "over the age of 21" credential may be useful when gaining
+access to a bar. If an issuer makes the credential have a very short expiration 
+date and an automatic renewal mechanism, then they could possibly correlate
+the holder's alcohol consumption behavior in a way that negatively impacts the
```

Comment:
Change "correlate" to "track".

Strike `alcohol consumption` and just have it say "holder's behavior".

Reply:
Done.